config/config.go

```diff
@@ -335,6 +341,7 @@ type Config struct {
 	GenesisState string                       `toml:"genesis_state" comment:"path to the genesis state file, relative to the root directory"`
 	Migrations   MigrationConfig              `toml:"migrations" comment:"zero downtime migration configuration"`
 	Checkpoint   Checkpoint                   `toml:"checkpoint" comment:"checkpoint info for the leader to sync to before proposing a new block"`
+	Erc20Bridge  ERC20BridgeConfig            `toml:"erc20_bridge" comment:"ERC20 bridge configuration"`
```

Comment:
This config is used in two places:
- evm-sync extension
- erc20-bridge extension

There are few places like oracle still use the general extension config, which I'm not sure if we should move them under `erc20_bridge` cofiguration.